lib/grammers-client/src/lib.rs

```diff
@@ -46,3 +46,9 @@ pub(crate) mod utils;
 
 pub use client::{Client, Config, InitParams, SignInError};
 pub use types::{button, reply_markup, ChatMap, InputMessage, Update};
+
+pub use grammers_crypto;
+pub use grammers_mtproto;
```

Comment:
I don't think we need to re-export all from `mtproto`. Realistically, one only needs the `::transport`s from it.

Reply:
So, `pub use grammers_mtproto::transport as mtproto_transport`? Although I think it would be better to just reexport the entire crate. How about `pub mod reexports { ... }`?

Reply:
The underlying transport protocols one can use in the client are the same as those the MTP supports. So I think it'd be fine to simply `pub use grammers_mtproto::transport`.

In user's code, `use grammers_client::transport::Intermediate` reads fine to me.

I don't think the concept of transports is used for anything else in the high-level API.